tdrs-frontend/src/mirage.js

```diff
@@ -26,6 +25,8 @@ export default function startMirage(
           return FAILED_AUTH_CHECK_DATA
         }
       })
+      this.post('/logs/', () => {})
```

Comment:
We were also getting network errors from this end point because the app calls it but we had no mock